src/modules/75-cd/components/ServicePopoverCard/ServicePopoverCard.tsx

```diff
@@ -16,37 +17,53 @@ export interface ServicePopoveCardProps {
 
 export function ServicePopoverCard(props: ServicePopoveCardProps): React.ReactElement {
   const { service } = props
+  const { getString } = useStrings()
 
   return (
     <div className={css.main}>
       <String tagName="div" className={css.title} stringID="primaryArtifactText" />
       {service?.artifacts?.primary ? (
-        <String
-          tagName="div"
-          stringID="artifactDisplay"
-          useRichText
-          vars={{
-            image: (service.artifacts.primary as unknown as any).imagePath,
-            tag: (service.artifacts.primary as unknown as any).tag
-          }}
-        />
+        <>
+          {!!(service.artifacts.primary as unknown as any).image && (
+            <Text color="grey800" font={'small'}>
+              {getString('imageLabel')}: {(service.artifacts.primary as unknown as any).image}
+            </Text>
+          )}
+          {!!(service.artifacts.primary as unknown as any).tag && (
+            <Text color="grey800" font={'small'}>
+              {getString('common.artifactTag')}: {(service.artifacts.primary as unknown as any).tag}
+            </Text>
+          )}
+          {!!(service.artifacts.primary as unknown as any).version && (
+            <Text color="grey800" font={'small'}>
+              {getString('version')}: {(service.artifacts.primary as unknown as any).version}
+            </Text>
+          )}
+        </>
       ) : (
         <String tagName="div" stringID="noArtifact" />
       )}
       {service?.artifacts?.sidecars && service.artifacts.sidecars.length > 0 ? (
         <>
           <String tagName="div" className={css.title} stringID="sidecarsText" />
           {service.artifacts.sidecars.map((artifact, index) => (
-            <String
-              tagName="div"
-              key={index}
-              stringID="artifactDisplay"
-              useRichText
-              vars={{
-                image: (artifact as unknown as any).imagePath,
-                tag: (artifact as unknown as any).tag
-              }}
-            />
+            <div key={index}>
+              {!!(artifact as unknown as any).image && (
+                <Text color="grey800" font={'small'}>
+                  {getString('imageLabel')}: {(artifact as unknown as any).image}
+                </Text>
+              )}
+              {!!(artifact as unknown as any).tag && (
+                <Text color="grey800" font={'small'}>
+                  {getString('common.artifactTag')}: {(artifact as unknown as any).tag}
+                </Text>
+              )}
+              {!!(artifact as unknown as any).version && (
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9238322)
```suggestion
              {!!(artifact as unknown as unknown).version && (
```